app/timetables/admin.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib import admin
 
 from .models import (Weekday, MealOption, Course, Meal, Timetable,
```

Comment:
Would it be a better idea to import the entire module here, since this module relies on several classes from it?